src/unstable-temp/DownloadImageModal/DownloadImageModal.tsx

```diff
@@ -72,6 +72,7 @@ export interface UnstableTempDownloadImageModalProps {
 	initialOsVersions?: OsVersionsByDeviceType;
 	isInitialDefault?: boolean;
 	downloadUrl: string;
+	onDownloadStart?: (onDownloadStart: boolean) => void;
```

Comment:
```suggestion
	onDownloadStart?: (downloadConfigOnly: boolean, downloadOptions: ???) => void;
```

Reply:
@thgreasi, I'm ok with the rename `downloadConfigOnly`, but I would not add a `downloadOptions` untill the refactor, since these are all splitted info like `appId`, `deviceType.slug` or `rawVersion`